### PR TITLE
[Components with more than 1 service] LPS-176895, LPS-177028, LPS-177035, LPS-177025, LPS-177328

### PR DESCRIPTION
**Pull request by RenanUmV**
[LPS-176895](https://issues.liferay.com/browse/LPS-176895), 
[LPS-177028](https://issues.liferay.com/browse/LPS-177028), 
[LPS-177035](https://issues.liferay.com/browse/LPS-177035), 
[LPS-177025](https://issues.liferay.com/browse/LPS-177025),
[LPS-177328](https://issues.liferay.com/browse/LPS-177328)

### CODE REVIEW

**Comment by RenanUmV** [comment]
ci:test:search

**Comment by RenanUmV** [comment]
ci:test:startup

**Comment by RenanUmV** [comment]
ci:test:commerce

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-inventory-impl/src/main/java/com/liferay/headless/commerce/admin/inventory/internal/resource/v1_0/WarehouseOrderTypeResourceImpl.java [hunk 1]
```diff
@@ -236,7 +236,7 @@ private WarehouseOrderType _toWarehouseOrderType(
 	private DTOConverterRegistry _dtoConverterRegistry;
 
 	@Reference(
-		target = "(component.name=com.liferay.headless.commerce.admin.inventoryinternal.dto.v1_0.WarehouseOrderTypeDTOConverter)"
+		target = "(component.name=com.liferay.headless.commerce.admin.inventory.internal.dto.v1_0.WarehouseOrderTypeDTOConverter)"
```
Comment (rafaprax) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-inventory-impl/src/main/java/com/liferay/headless/commerce/admin/inventory/internal/resource/v1_0/WarehouseOrderTypeResourceImpl.java:239]: Hey @RenanUmV, this change should not be here. Please fixup this with the proper commit. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by RenanUmV** [comment]
ci:test:sf

**Comment by zhangyan0701** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:search

**Review by rafaprax — APPROVED** [review]

**Comment by catarineaguiar** [comment]
ci:stop

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
ci:stop

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
ci:test:commerce

**Pull request closed without merging**